### PR TITLE
Games

### PR DESCRIPTION
**Pull request by DrunkenToast**
Move games to seperate file and implement some others

### CODE REVIEW

**Comment by DrunkenToast** [comment]
Review for merge

File: src/games.ts [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+    if (!buzzCollector) return;
+    if (!controlCollector) return;
+    
+    // Collectors
+    buzzCollector.on('collect', i => {
+      buzzCollector.resetTimer();
+      controlCollector.resetTimer();
+      
+      if (this.buttonBuzzer[0].disabled) return;
```
Comment (PandaGamersHDNL) [inline src/games.ts]: impossible to collect a button if the button is disabled

Comment (DrunkenToast) [reply src/games.ts]: I know, was extra safety check.

File: src/games.ts [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+      if (!(i.member instanceof GuildMember)) return;
+
+      if (!this.usersBuzzed.includes(i.member)) {
+        this.usersBuzzed.push(i.member);
```
Comment (PandaGamersHDNL) [inline src/games.ts:53]: work with id

Comment (DrunkenToast) [reply src/games.ts:53]: This array gets accessed later for usernames

**Review by PandaGamersHDNL — APPROVED** [review]

**Pull request merged**